brickstrap.sh

```diff
@@ -404,7 +440,10 @@ case "${cmd}" in
     copy-root)           copy-root;;
     configure-packages)  configure-packages;;
     run-hook)            run-hook ${BOARDDIR}/hooks/${run_hook_arg};;
-    run-hooks)           run-hooks;;
+    run-hooks)
+        run-hooks
+        dump-info-hook
```

Comment:
Makes more sense to me too. :wink:

Although we don't want it tacked onto the `run-hooks` command. Like the example that you gave, if you want to do a checksum on the image file, the image file has not been created at this point. Instead, I think it should be a new stand-alone command (and be the final step of the `all` command). And maybe have a more generic name.